tests/Unit/Utilities/Test_Math.cpp

```diff
@@ -23,6 +24,21 @@ void test_smoothstep() {
   CHECK_ITERABLE_APPROX(
       smoothstep<N>(-1., 1., DataVector({-2., -1., 0., 1., 2.})),
       DataVector({0., 0., 0.5, 1., 1.}));
+  if constexpr (N > 0) {
+    INFO("Test smoothstep_deriv");
+    CHECK(smoothstep_deriv<N>(0., 1., 0.) == approx(0.));
+    CHECK(smoothstep_deriv<N>(0., 1., 0.) == approx(0.));
```

Comment:
These two checks are the same... is the second one supposed to evaluate at arg=1 instead of arg=0?
```suggestion
    CHECK(smoothstep_deriv<N>(0., 1., 1.) == approx(0.));
```